src/java/htsjdk/samtools/CRAMIterator.java

```diff
@@ -49,7 +49,7 @@
     private ArrayList<SAMRecord> records;
     private SAMRecord nextRecord = null;
     @SuppressWarnings({"CanBeFinal", "FieldCanBeLocal"})
-    private boolean restoreNMTag = true;
+    private boolean restoreNMTag = false;
     @SuppressWarnings({"CanBeFinal", "FieldCanBeLocal"})
     private boolean restoreMDTag = false;
```

Comment:
Both restoreNMTag and restoreMDTag, and their corresponding SuppressWarnings annotations are no longer used and can be removed.